ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/channels/WindowedSeekableByteChannel.java

```diff
@@ -124,7 +118,13 @@ public long size() throws IOException {
     public SeekableByteChannel truncate(final long size) throws IOException {
         throw new TruncateNotAllowedException();
     }
-    
+
+    private void checkClosed() {
```

Comment:
The one thing that I could do is change the check to ask the wrapped channel.  There isn't a parent per-say which is why I added the logic in here.